app/views/core_induction_programmes/years/_module_list.html.erb

```diff
@@ -31,7 +31,7 @@
           <hr class="govuk-section-break govuk-section-break--visible" />
           <div class="app-task-list__section">
             <div class="app-task-list__flex">
-              <span><%= course_module.self_study_lessons.count.humanize.capitalize %> self-study <%= "session".pluralize(course_module.course_lessons.count) %></span>
+              <span><%= course_module.self_study_lessons.count %> self-study <%= "session".pluralize(course_module.course_lessons.count) %></span>
```

Comment:
I think this was the only place where we used `humanize` gem - can we remove it from our gemfile and from our project in general?